src/app/rundown-view/components/rundown-header-context-menu/rundown-header-context-menu.component.ts

```diff
@@ -65,6 +99,14 @@ export class RundownHeaderContextMenuComponent {
     this.rundownService.activate(this.rundown.id).subscribe()
   }
 
+  public openRehearseRundownDialog(): void {
+    this.dialogService.createConfirmDialog(this.rundown.name, 'Are you sure you want to rehearsal the Rundown?', 'Rehearse', () => this.rehearsalRundown())
+  }
+
+  public rehearsalRundown(): void {
```

Comment:
This is a method name so it should use the verb `rehearse`

Reply:
Fixed